src/utils/AxiosFactory.ts

```diff
@@ -30,22 +30,30 @@ export default class AxiosFactory {
     if (!axiosInstance) {
       // Create
       axiosInstance = axios.create(instanceConfiguration.axiosConfig);
+      // FIX ME
+      // Error Message: Converting circular structure to JSON
+      // in some cases
+      //
       // Add a Request interceptor
-      axiosInstance.interceptors.request.use((request: AxiosRequestConfig) => {
-        Logging.logAxiosRequest(tenantID, request);
-        return request;
-      }, async (error: AxiosError) => {
-        Logging.logAxiosError(tenantID, error);
-        return Promise.reject(error);
-      });
+      // axiosInstance.interceptors.request.use((request: AxiosRequestConfig) => {
+      //   Logging.logAxiosRequest(tenantID, request);
+      //   return request;
+      // }, async (error: AxiosError) => {
+      //   Logging.logAxiosError(tenantID, error);
+      //   return Promise.reject(error);
+      // });
+      //
+      // FIX ME
+      // Error Message: Converting circular structure to JSON
+      //
       // Add a Response interceptor
-      axiosInstance.interceptors.response.use((response: AxiosResponse) => {
-        Logging.logAxiosResponse(tenantID, response);
-        return response;
-      }, async (error: AxiosError) => {
-        Logging.logAxiosError(tenantID, error);
-        return Promise.reject(error);
-      });
+      // axiosInstance.interceptors.response.use((response: AxiosResponse) => {
+      //   Logging.logAxiosResponse(tenantID, response);
+      //   return response;
+      // }, async (error: AxiosError) => {
+      //   Logging.logAxiosError(tenantID, error);
+      //   return Promise.reject(error);
+      // });
```

Comment:
That's another breaking change that will forbid the PR merge. It needs to be fixed before.

Reply:
I changed it back.
The errors for OICP requests will reappear **Error: "Converting circular structure to JSON"**
I created a copy of the AxiosFactory.ts for the OICP requests. I use this copy (AxiosFactoryOICP.ts) until we fixed the issue

Reply:
I think you can also add a boolean to bypass that code if needed to the main class.

Reply:
okay